test/test_deque.jl

```diff
@@ -200,4 +200,26 @@
         @test isempty(q)
     end
 
+    @testset "show" begin
+        q = Deque{Int}()
+        intstr = sprint(show, Int)
+        @test sprint(show, q) == "0-element Deque{$(intstr)}"
+        @test sprint(show, q.head) == "0-element DataStructures.DequeBlock{$(intstr)}(capa = 1024, front = 1, back = 0)"
+        push!(q, 1)
+        @test sprint(show, q) == "1-element Deque{$(intstr)}:\n 1"
+        @test sprint(show, q.head) == "1-element DataStructures.DequeBlock{$(intstr)}(capa = 1024, front = 1, back = 1):\n 1"
+        for i in 2:50
+            push!(q, i)
+        end
+        @test sprint(show, q) == "50-element Deque{$(intstr)}:\n  1\n  2\n  3\n  4\n  5\n  6\n  7\n  8\n  9\n 10\n 11\n 12\n 13\n 14\n 15\n 16\n 17\n 18\n 19\n 20\n 21\n 22\n 23\n 24\n 25\n 26\n 27\n 28\n 29\n 30\n 31\n 32\n 33\n 34\n 35\n 36\n 37\n 38\n 39\n 40\n 41\n 42\n 43\n 44\n 45\n 46\n 47\n 48\n 49\n 50"
```

Comment:
What about using triple-quoted string? https://pkg.julialang.org/docs/julia/THl1k/1.1.0/manual/strings.html#Triple-Quoted-String-Literals-1

It could improve readability of tests.